pkg/pool-weighted/contracts/smart/ManagedPool.sol

```diff
@@ -164,6 +165,7 @@ contract ManagedPool is BaseWeightedPool, ProtocolFeeCache, ReentrancyGuard {
         uint256 protocolSwapFeePercentage;
         uint256 managementSwapFeePercentage;
         uint256 managementAumFeePercentage;
+        IAumProtocolFeesCollector aumProtocolFeesCollector; // Should be a constructor parameter, but blew up the stack
```

Comment:
```suggestion
        IAumProtocolFeesCollector aumProtocolFeesCollector;
```

Reply:
I found this comment misleading. If anything we could have a comment above the struct saying 'the pool receives so many arguments that it runs into stack-too-deep errors, so they are passed in a memory struct instead to avoid those' or something like that.

Reply:
I'd like to group the "factory" parameters (vault, owner, and the temporarily pausable parameters) into a separate struct to keep NewPoolParams pure, but just updating the comment now in the interest of time.